modules/mbedtls/Kconfig.tls-generic

```diff
@@ -488,6 +488,7 @@ config MBEDTLS_PSA_CRYPTO_C
 
 config MBEDTLS_USE_PSA_CRYPTO
 	bool "Use PSA APIs instead of legacy MbedTLS when possible"
+	default y if MBEDTLS_PSA_CRYPTO_C
```

Comment:
This should be MBEDTLS_PSA_CRYPTO_CLIENT

Reply:
Correct, thanks! Fixed